2017/hipchat-breakin.md

```diff
@@ -61,11 +61,11 @@ Kommt es zum Ausfall einer Plattform hilft kein [Service-Level Agreement][sla] (
 mit welchem der Anbieter seine Zuverlässigkeit gegenüber dem Kunden verspricht.
 Dann sind die Systeme halt aus, nicht erreichbar oder in sonstigen Zuständen – der
 Kunde hat hier keine Eingriffsmöglichkeiten. Es bleiben lediglich rechtliche
-Schritte, wie Rückzahlungen oder Entschädigungen. Steht mein Geschäft aber wegen
+Schritte, wie Rückzahlungen oder Entschädigungen. Steht das eigene Geschäft aber wegen
 Ausfall für mehrere Tage still, ist ein unverschuldeter finanzieller Verlust unumgänglich.
 
 Noch schlimmer als ein Ausfall ist ein rufschädigender Verlust von
-Kundendaten im eigenen Geschäft. Nutzt eine Firma z.B. ein cloud-basiertes [CRM-System][crm]
+Kundendaten des eigenen Betriebs. Nutzt eine Firma z.B. ein Cloud-basiertes [CRM-System][crm]
 zur Verwaltung von Kunden und Aufträgen, und die Kundendaten dieser SaaS-Plattform
```

Comment:
Siehe `[stallman-saas]: https://www.gnu.org/philosophy/who-does-that-server-really-serve.html` im letzen Teil.